src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1643,81 +1649,112 @@ public boolean isLossless(String codecA) {
+			}
+			result.append(", Video Tracks: ").append(getVideoTrackCount());
+			result.append(", Video Codec: ").append(getCodecV());
+			if (getDuration() != 0) {
+				result.append(", Duration: ").append(getDurationString());
+			}
+			result.append(", Video Resolution: ").append(getWidth()).append(" x ").append(getHeight());
+			if (isNotBlank(getFrameRate())) {
+				result.append(", Frame Rate: ").append(getFrameRate());
+			}
+			if (isNotBlank(getFrameRateOriginal())) {
+				result.append(", Original Frame Rate: ").append(getFrameRateOriginal());
+			}
+			if (isNotBlank(getFrameRateMode())) {
+				result.append(", Frame Rate Mode: ");
+				result.append(getFrameRateModeRaw());
+				if (isNotBlank(getFrameRateModeRaw())) {
+					result.append(" (").append(getFrameRateModeRaw()).append(")");
+				}
+			} else if (isNotBlank(getFrameRateModeRaw())) {
+				result.append(", Frame Rate Mode Raw: ");
+				result.append(getFrameRateModeRaw());
+			}
+			if (isNotBlank(getMuxingMode())) {
+				result.append(", Muxing Mode: ").append(getMuxingMode());
+			}
+			if (isNotBlank(getMatrixCoefficients())) {
+				result.append(", Matrix Coefficients: ").append(getMatrixCoefficients());
+			}
+			if (isNotBlank(avcLevel)) {
+				result.append(", AVC Level: ").append(getAvcLevel());
+			}
+//			if (isNotBlank(getHevcLevel())) {
+//				result.append(", HEVC Level: ");
+//				result.append(getHevcLevel());
+			if (getVideoBitDepth() != 8) {
+				result.append(", Video Bit Depth: ").append(getVideoBitDepth());
+			}
+			if (isNotBlank(getFileTitleFromMetadata())) {
+				result.append(", File Title from Metadata: ").append(getFileTitleFromMetadata());
+			}
+			if (isNotBlank(getVideoTrackTitleFromMetadata())) {
+				result.append(", Video Track Title from Metadata: ").append(getVideoTrackTitleFromMetadata());
+			}
 
-		if (thumb != null) {
-			result.append(", thumb size: ");
-			result.append(thumb.length);
-		}
-		if (isNotBlank(muxingMode)) {
-			result.append(", muxing mode: ");
-			result.append(muxingMode);
-		}
+			if (getAudioTrackCount() > 0) {
+				appendAudioTracks(result);
+			}
 
-		result.append(", mime type: ");
-		result.append(mimeType);
+			if (hasSubtitles()) {
+				appendSubtitleTracks(result);
+			}
 
-		if (isNotBlank(matrixCoefficients)) {
-			result.append(", matrix coefficients: ");
-			result.append(matrixCoefficients);
+		} else if (getAudioTrackCount() > 0) {
+			if (getBitrate() != 0) {
+				result.append(", Bitrate: ").append(getBitrate());
+			}
+			if (getDuration() != 0) {
+				result.append(", Duration: ").append(getDurationString());
+			}
+			appendAudioTracks(result);
 		}
-
-		if (isNotBlank(avcLevel)) {
-			result.append(", avc level: ");
-			result.append(avcLevel);
+		if (getImageCount() > 0) {
+			if (getImageCount() > 1) {
+				result.append(", Images: ").append(getImageCount());
+			}
+			result.append(", Image Width: ").append(getWidth());
+			if (getHeight() != 0) {
```

Comment:
I disagree with this - DNL is extremely rare, and I've implemented it in Metadata-extractor so that in a future version it will still be parsed with height. I'd say that it would be very useful to get 0 as height, as that's very likely to be a bug.